### ISSUE TITLE
[VoiceOver] Media Library concerns

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-iOS#12954]
## Media Library

<img src="https://user-images.githubusercontent.com/198826/68981199-e5933800-07bf-11ea-848d-fa900d2f5f75.png" width="320">

### ~[A1] The _Preview_ and _Insert_ buttons should be visible but disabled by default~

~They are not discoverable if they are hidden until you select an image. It may be difficult for users to figure out how to interact with the UI.~

~The behavior of hiding and showing also interrupts with the scrolling dictation like:~ 

~1. Scroll down. ”Page 6 of 7”~
~2. Scroll down. “Page 7 of 7”~
~3. Select an image.~ 
~4. Scroll up. “Page 7 of 8”.~ 

~The number of pages _changed_ even though the data didn't change.~ 

~The buttons can also make use of `.accessibilityValue` if they don't have that already.~ 

### [A2] Buttons are missing labels

- [ ] The _Cancel_ button shown after pressing _Edit_ can probably be better labeled as “Cancel editing”.
- [ ] The _Trash_ button shown when editing doesn't have a label. VoiceOver just says “Dimmed. Button”

## Media Library → Image Details

<img src="https://user-images.githubusercontent.com/198826/68981216-edeb7300-07bf-11ea-975c-64a79be5dd3c.png" width="320">

- [ ] The _Trash_ and _Share_ button have no labels
- [ ] The image has no `.image` trait

**Comment by rohanrk** [comment]
This is quite closely related to the PR I just put out. I'm happy to work on this once that PR is closed

**Comment by shiki** [comment]
@rohanrk Cool! I assigned you to the task. 🙂

**Comment by rohanrk** [comment]
Could I get some more info on the preview and insert buttons? I'm not quite sure what they refer to in this context. I think preview has to do with previewing selected images but this is handled through a long press rather than a button.

**Comment by shiki** [comment]
It looks like the UI has changed a bit and it's no longer possible to pick multiple images. I have crossed out the ones the issues that are no longer valid. 

Sorry about that, @rohanrk!

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
